frontend/entities/Types.md

```diff
@@ -0,0 +1,8 @@
+# Types
+
+The standard used for `Typescript` is the following:
+
+1. Every type should start with `T` e.g `TImage`.
+2. Model types should have the same name as the factory e.g. `User`.
+3. Components should define the prop types with `TProps`.
+4. Instances should be only used when there is a feature not supported by `Types`.
```

Comment:
I would consider adding an example of what the type for Image might look like here, to be consistent with the structure of the other entity files in this folder.